src/ray/common/status.h

```diff
@@ -313,6 +313,8 @@ class RAY_EXPORT Status {
   // Returns the string "OK" for success.
   std::string ToString() const;
 
+  std::string StatusString() const { return ToString(); }
```

Comment:
What's the difference between StatusString and ToString

Reply:
Functionality-wise no difference, the reason to have this function is for pretty print status for GTEST macros.

In nuro, we have another way to handle:
```
template <typename Status>
struct ConsumeStatus {...};

template <>
struct ConsumeStatus<Status> {...};
template <>
struct ConsumeStatus<StatusOr> {...};
```
Having a `StatusString` is easier to implement.

Reply:
Could you elaborate, I don't think I get it. Why cannot the caller call `ToString()`?

Reply:
We use it in the macro
```
CHECK_OK(s) << s.StatusString();
```
it's a macro thus duck typing, so we have to have the same function name for both `Status` and `StatusOr`

Reply:
Calling `ToString` is OK for `Status`, but improper for `StatusOr`, because we haven't implement the stringify function for value-type

Reply:
Why need a comment here?

Reply:
Anyway I added a comment for necessity from duck-type template/macro.